go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -96,100 +93,80 @@ func newEgressFirewallRule(rawEgressFirewallRule egressfirewallapi.EgressFirewal
 
 // NOTE: Utilize the fact that we know that all egress firewall related setup must have a priority: types.MinimumReservedEgressFirewallPriority <= priority <= types.EgressFirewallStartPriority
 func (oc *Controller) syncEgressFirewall(egressFirwalls []interface{}) {
+	// Lookup all ACLs used for egress Firewalls
+	egressFirewallACLLookupFcn := func(item *nbdb.ACL) bool {
+		return (item.Priority <= types.EgressFirewallStartPriority ||
+			item.Priority >= types.MinimumReservedEgressFirewallPriority)
+	}
+	egressFirewallACLs, err := libovsdbops.FindACLs(oc.modelClient.GetClient(), egressFirewallACLLookupFcn)
```

Comment:
Nothing against specifying a functor, but prefer `FindACLsByPriority` as it use a commonly used criteria, or otherwise `FindACLsByPredicate`.

Reply:
Done